docs/development-setup.md

```diff
@@ -17,8 +17,8 @@ We'll use the `oc cluster up` method, since it's very easy to use:
     ```
     $ oc cluster up
     ```
-    (If you are running into some connection issues, please flush your iptables
-    rules bufore starting the cluster with `iptables -F`)
```

Comment:
Clearly, 'oc cluster' doesn't work very well with a laptop with a dynamic IP and Fedora's default firewall rules. (Origin 3.6 has some patches in it that are supposed to improve the dynamic IP situation.) I felt that things went a bit better when I followed the instructions in that document, and stopped removing all the iptables rules, restarting docker, restarting firewalld, etc. I'll do a bit more experimentation and see if I can figure out some wording that suggests 'iptables -F' as another thing to try.